### ISSUE TITLE
travis-CI is failing

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/curatedMetagenomicData#105]
```
$ R CMD build --keep-empty-dirs --no-resave-data --no-vignettes .
Error: '--no-vignettes' is defunct:
  use '--no-build-vignettes' instead
Execution halted
The command "R CMD build --keep-empty-dirs --no-resave-data --no-vignettes ." failed and exited with 1 during .
Your build has been stopped.
```

**Comment by LiNk-NY** [comment]
Fixed in 8e58757a918d9a638a4e6ad7e23f46cf1ba660cb

**Comment by lwaldron** [comment]
still failing, seems to be hung for 3 hours on:

```
trying URL 'https://cloud.r-project.org/src/contrib/bit64_0.9-7.tar.gz'
Content type 'application/x-gzip' length 125539 bytes (122 KB)
==================================================
downloaded 122 KB
Installing bit64
```
